common/tools/dev-tool/src/commands/run/testVitest.ts

```diff
@@ -90,6 +91,13 @@ export default leafCommand(commandInfo, async (options) => {
   try {
     if (options["test-proxy"]) {
       if (options["test-proxy-debug"]) process.env["Logging__LogLevel__Default"] = "Debug";
+
+      // restore recordings first in CI to avoid impacting the first playback test
+      if (process.env["BUILD_BUILDNUMBER"]) {
```

Comment:
Do you ever hit the now-5-second hook timeout when running locally?

I'm a bit concerned that by dropping hook timeout from 50 to 5 seconds, that the test needing to restore _locally_ may hit issues.

Reply:
> I'm a bit concerned that by dropping hook timeout from 50 to 5 seconds, that the test needing to restore _locally_ may hit timeout issues.

I bumped it from default 10 seconds to 50 seconds to unblock CI and now I am changing it back to default.  We couldn't repro locally.